### PR TITLE
LANG-1175: removed ant-based build

### PR DESCRIPTION
**Pull request by wikier**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -4,6 +4,7 @@ target
 maven-eclipse.xml
 build.properties
 site-content
+*~
```
Comment (britter) [inline .gitignore]: What does this mean?

File: .gitignore [hunk 2]
```diff
@@ -4,6 +4,7 @@ target
 maven-eclipse.xml
 build.properties
```
Comment (britter) [inline .gitignore]: Can be dropped from .gitignore

File: src/changes/changes.xml [hunk 3]
```diff
@@ -22,6 +22,7 @@
   <body>
 
   <release version="3.5" date="tba" description="tba">
+    <action issue="LANG-1175" type="fix" dev="wikier" dute-to="Benedikt Ritter">Remove Ant-based build</action>
```
Comment (britter) [inline src/changes/changes.xml]: Copy and paste error. Should be `due-to`. I've corrected this for the other lines.

**Comment by britter** [comment]
Looks good to me. But we should also drop:
* default.properties
* src/release-tools (it also depends on the main ant build)

File: .gitignore (hunk 1 above)
Comment (wikier) [inline .gitignore]: commonly some editors (e.g., emacs) use the suffix `~` to create backup files

**Comment by wikier** [comment]
updated from some comments by @britter

File: src/assembly/src.xml [hunk 4]
```diff
@@ -32,7 +32,6 @@
                 <include>pom.xml</include>
                 <include>PROPOSAL.html</include>
                 <include>RELEASE-NOTES.txt</include>
-                <include>build.xml</include>
                 <include>default.properties</include>
```
Comment (britter) [inline src/assembly/src.xml]: Should also be dropped

**Comment by britter** [comment]
Looks good to me now. Feel free to merge to master.

**Comment by wikier** [comment]
I don't have permissions on the main git repo, so please @britter merge it.

**Pull request merged**